contributed_definitions/nyaml/NXapm_paraprobe_config_distancer.yaml

```diff
@@ -1,453 +0,0 @@
-category: application
```

Comment:
github shows a removed file and a new one. Maybe a renaming step would have helped in reviewing the changes.

Reply:
Agree, I should have made a commit where I just renamed and making edits thereafter